Sources/ComposableArchitecture/Documentation.docc/Tutorials/BuildingSyncUps/05-PersistingSyncUps/PersistingSyncUps-01-code-0006.swift

```diff
@@ -0,0 +1,25 @@
+import ComposableArchitecture
+import SwiftUI
+
+@Reducer
+struct SyncUpsList {
+  // ...
+}
+
+struct SyncUpsListView: View {
+  // ...
+}
+
+#Preview {
+  @Shared(.syncUps) var syncUps = [.mock]
+  NavigationStack {
+    SyncUpsListView(
+      store: Store(
+        initialState: SyncUpsList.State()
+      ) {
+        SyncUpsList()
+          ._printChanges()
+      }
+    )
+  }
```

Comment:
📓 In this new step, I've updated the preview to setup a Shared props outside of the SyncUpsList state init.